DESCRIPTION

```diff
@@ -53,7 +53,7 @@ VignetteBuilder:
 RdMacros:
     lifecycle
 Config/Needs/verdepcheck: rstudio/bslib, mllg/checkmate,
-    davidgohel/flextable, cran/grid, rstudio/htmltools, yihui/knitr,
```

Comment:
grid is removed from cran right? this package is in import

Reply:
This is a base package. The `cran/grid` is 22 years old and it will break this

Reply:
I mena, it will break this if we add the github mirror as it's not the "real" code we want to test

Reply:
oh okay got it.